git-hooks/pre-commit

```diff
@@ -10,7 +10,13 @@ exec 1>&2
 
 changed_files=$(git diff --cached --name-only)
 changed_go_files=$(git diff --cached --name-only | grep -E '.go$' || true)
-all_go_files=$(find -name '*.go' | grep -v /vendor/)
+
+find_cmd="find"
```

Comment:
I think this is quite environment specific - for example on my system I have find but not gfind.

Perhaps better to check if `find` exists and if not try `gfind`.